node-notes-cli/add.js

```diff
@@ -0,0 +1,14 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+module.exports = function add(addition) {
+  var newEntry = addition;
+  var newKey = data.nextId;
+  data.notes[newKey] = newEntry;
+  data.nextId = data.nextId + 1;
+  JSON.stringify(data, null, 2);
+  fs.writeFile('data.json', JSON.stringify(data), function (err) {
```

Comment:
The first `JSON.stringify` call is redundant and doesn't do anything useful. Please remove it.